backend/src/main/java/com/example/backend/controller/ExchangeController.java

```diff
@@ -36,8 +37,8 @@ public ExchangeCard getByID(@PathVariable String id) {
     }
 
     @PutMapping(path = "/{id}")
-    public ExchangeCard updateEntry(@PathVariable String id, @RequestPart(value = "entry") String entryToUpdate, @RequestPart(value = "file") MultipartFile cardImage) throws IOException {
-        return exchangeService.updateEntry(id, entryToUpdate, cardImage);
+    public ExchangeCard updateEntry(@PathVariable String id, @RequestPart(value = "entry") String entryToUpdate, @RequestPart(value = "file") MultipartFile cardImage, String author) throws IOException {
```

Comment:
Where does the author Parameter come from? There ist no @RequestBody or any other Annotation?